jobfunnel/config/settings.py

```diff
@@ -0,0 +1,160 @@
+"""Settings YAML Schema w/ validator
+"""
+import ipaddress
+
+from cerberus import Validator
+
+from jobfunnel.resources import (LOG_LEVEL_NAMES, DelayAlgorithm, Locale,
+                                 Provider)
+from jobfunnel.resources.defaults import *
+
+SETTINGS_YAML_SCHEMA = {
```

Comment:
Really like this too. Having a validator like this will force us to have a standard for our settings files.